tests/profiler/test_rule_state.py

```diff
@@ -12,38 +16,128 @@
 from great_expectations.profiler.rule.rule_state import RuleState
 
 
+# noinspection PyPep8Naming
+@pytest.fixture
+def column_Age_storage_type_domain():
+    return Domain(
+        domain_kwargs={"column": "Age", "batch_id": "1234567890"},
+        domain_type=StorageDomainTypes.COLUMN,
+    )
+
+
+# noinspection PyPep8Naming
+@pytest.fixture
+def column_Date_storage_type_domain():
+    return Domain(
+        domain_kwargs={"column": "Date", "batch_id": "1234567890"},
+        domain_type=StorageDomainTypes.COLUMN,
+    )
+
+
+@pytest.fixture
+def single_syllable_parameter_container():
```

Comment:
What does the word `syllable` mean in this context?

Reply:
@talagluck Good question -- if the parameter name that follows the dollar sign is `my.fancy.incredible.wonderful.default.date_format` then each word, separated by the period, is termed a `syllable`.  Thank you.